src/Serilog.Sinks.Datadog.Logs/Configuration/Extensions/Microsoft.Extensions.Configuration/LoggerConfigurationDatadogLogsExtensions.cs

```diff
@@ -53,7 +53,7 @@ public static class LoggerConfigurationDatadogExtensions
             TimeSpan? batchPeriod = null,
             int? queueLimit = null,
             Action<Exception> exceptionHandler = null,
-            bool detectTCPDisconnection = false)
+            bool detectTCPDisconnection = false, IDatadogClient client = null)
```

Comment:
I couldn't find anything about this project's policy on breaking changes, so I just wanted to note that even removing/adding an optional parameter is a binary breaking change as the method signature changes.

The correction of `formatMessage` ->`FormatMessage` below is both a binary and source breaking change.

Reply:
Thanks for calling this out. We currently don't have a strict policy on breaking changes between versions. It is fairly common to add parameters to this method between versions however. When we publish a release we will call out the major changes for awareness.